lib/src/controller.dart

```diff
@@ -91,6 +93,11 @@ class TimetableController<E extends Event> {
   }) =>
       animateTo(LocalDate.today(), curve: curve, duration: duration);
 
+  Future<void> animateToInitialDate({
+    Curve curve = Curves.easeInOut,
+    Duration duration = const Duration(milliseconds: 200),
+  }) => animateTo(initialDate, curve: curve, duration: duration);
```

Comment:
I don't think that `animateToInitialDate` and `jumpToInitialDate` are useful in general, and since these calls can easily be done manually, I'd rather not add them

Reply:
ok